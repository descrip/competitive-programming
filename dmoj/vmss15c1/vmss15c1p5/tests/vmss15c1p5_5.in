8 973 977
applecomputer 812 19 22
dp 678 24 14
fuji 925 18 17
green 9 11 33
icecream 833 7 12
morris 261 29 29
mountain 981 28 1
thisisaname 758 2 5
