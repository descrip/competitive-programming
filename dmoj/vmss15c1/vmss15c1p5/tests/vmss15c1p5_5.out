165379
applecomputer 0
dp 0
fuji 0
green 0
icecream 0
morris 0
mountain 21
thisisaname 191
